src/authDevRouter.js

```diff
@@ -0,0 +1,73 @@
+const router = require('express').Router()
```

Comment:
what's happening with this file? Do we want it kept/should we put it somewhere else